ores/utilities/label_reverted.py

```diff
@@ -5,11 +5,11 @@
 <rev_id>\t<reverted>
 
 Usage:
-    features_reverted -h | --help
-    features_reverted --api=<url> [--revert-radius=<revs>]
-                                  [--revert-window=<secs>]
-                                  [--rev-pages=<path>]
-                                  [--rev-reverteds=<path>]
+    label_reverted -h | --help
+    label_reverted --api=<url> [--revert-radius=<revs>]
```

Comment:
What about the [ipython/Enwiki Revert Classifier.ipynb](https://github.com/halfak/Objective-Revision-Evaluation-Service/blob/d954cf8fc48b8ead065134184d6644ccff939cbb/ipython/Enwiki%20Revert%20Classifier.ipynb#L30-L31)? Shouldn't it be updated too?